scripts/ci-footprint-and-performance/tests/010-hwlatdetect.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=scripts/ci-footprint-and-performance/tests/functions.sh
+source "${SCRIPTDIR}/functions.sh"
+
+TEST_DURATION=$(( 2 * 60 ))
+POD=hwlatdetect
+
+sudo tee /tmp/hwlatdetect.yaml << EOF
```

Comment:
Can we add a description on what we're running in the pod?

Reply:
Also, should we use mktemp for the file in /tmp?

Reply:
Added desc, but didn't use mktemp - it's annoying for manually iterating on the yaml